### PR TITLE
add  blog post about pip support

### PR DESCRIPTION
**Pull request by mireille-raad**
:) it is worth sharing on the blog. no? 

@coolreader18  are you okay with adding the asciinema links?

### CODE REVIEW

File: _posts/2021-01-27-pip-support.markdown [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+## get-pip.py + pip install requests
+[![asciicast](https://asciinema.org/a/fuTmP0wMD8HnvwsOkPRPuVHoa.svg)](https://asciinema.org/a/fuTmP0wMD8HnvwsOkPRPuVHoa)
+
+
+## pip installs itself and setup tools
+[![asciicast](https://asciinema.org/a/y2rIeqELmzn5LCaWI6MLMvujz.svg)](https://asciinema.org/a/y2rIeqELmzn5LCaWI6MLMvujz)
+
+
+## What you need to do
+This update is so fresh, things are not merged into the `master` or `release` branches yet. For now, to test this out, you have to merge `coolreader18/fix-http` `coolreader18/csv-writer` `coolreader18/misc-os` `coolreader18/name-mangling` `update-python-version` `coolreader18/empty-map-cformat`.
```
Comment (coolreader18) [inline _posts/2021-01-27-pip-support.markdown:25]:
```suggestion
This update is so fresh, things are not merged into the `master` or `release` branches yet. For now, to test this out, you have to checkout the `pip-merg` branch, and make sure to build with the `ssl` feature:
```
(I can't put a code block in cause it conflicts with the `suggestion` code block lol :grin: )
```sh
$ cd RustPython
$ git fetch --all
$ git checkout pip-merg
$ cargo install --path . --features ssl
$ curl https://bootstrap.pypa.io/get-pip.py -O
$ rustpython get-pip.py
...lots of work...
$ rustpython -m pip --version
pip 21.0 from /home/.../.local/lib/rustpython3.9/site-packages/pip (python 3.9)
```

**Review by coolreader18 — APPROVED** [review]
Yeah, I think that's totally great!

File: _posts/2021-01-27-pip-support.markdown (hunk 1 above)
Comment (mireille-raad) [reply _posts/2021-01-27-pip-support.markdown:25]: do you want me to commit the suggestion and then add this at the end?
```
$ cd RustPython
$ git fetch --all
$ git checkout pip-merg
$ cargo install --path . --features ssl
$ curl https://bootstrap.pypa.io/get-pip.py -O
$ rustpython get-pip.py
...lots of work...
$ rustpython -m pip --version
  pip 21.0 from /home/.../.local/lib/rustpython3.9/site-packages/pip (python 3.9)
```

Comment (coolreader18) [reply _posts/2021-01-27-pip-support.markdown]: Yeah, that'd be great :)

Comment (mireille-raad) [reply _posts/2021-01-27-pip-support.markdown]: done. looks good?

**Comment by mireille-raad** [comment]
done. looks good?

**Comment by coolreader18** [comment]
Yep!

**Pull request merged**